Sources/Apodini/Configurations/Application/Application+FCM.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  File.swift
+//  
+//
+//  Created by Tim Gymnich on 30.12.20.
+//
+
+import Foundation
+import FCM
+
+extension Application {
+    /// Firebase Cloud Messaging
+    public var fcm: FCM {
+        .init(application: self)
+    }
+
+    /// Firebase Cloud Messaging
+    public struct FCM {
+        struct ConfigurationKey: StorageKey {
+            // swiftlint:disable nesting
+            typealias Value = FCMConfiguration
+        }
+
+        /// FCM Configuration
+        public var configuration: FCMConfiguration? {
```

Comment:
We could rename the struct to `FCMApplication` or something else. This would allow us to import the `FCMConfiguration` from the library like this `FCM.FCMConfiguration`, which would solve the naming issue with our configuration struct `FCMConfiguration`.

Reply:
fixed it

Reply:
Setting `NotificationCenter.shared.application = app` inside of `Webservice` should mitigate setting the app twice